src/test/regress/sql/multi_simple_queries.sql

```diff
@@ -319,6 +319,16 @@ SELECT * FROM articles TABLESAMPLE BERNOULLI (0) WHERE author_id = 1;
 SELECT * FROM articles TABLESAMPLE SYSTEM (100) WHERE author_id = 1 ORDER BY id;
 SELECT * FROM articles TABLESAMPLE BERNOULLI (100) WHERE author_id = 1 ORDER BY id;
 
+SELECT count(*) FROM articles TABLESAMPLE SYSTEM (0);
+SELECT count(*) FROM articles TABLESAMPLE BERNOULLI (0);
+SELECT count(*) FROM articles TABLESAMPLE SYSTEM (100);
+SELECT count(*) FROM articles TABLESAMPLE BERNOULLI (100);
+
+SELECT count(*) FROM (SELECT random() FROM articles TABLESAMPLE SYSTEM (0)) a;
+SELECT count(*) FROM (SELECT random() FROM articles TABLESAMPLE BERNOULLI (0)) a;
+SELECT count(*) FROM (SELECT random() FROM articles TABLESAMPLE SYSTEM (100)) a;
+SELECT count(*) FROM (SELECT random() FROM articles TABLESAMPLE BERNOULLI (100)) a;
```

Comment:
I think this feature deserves a few more tests.

1. TABLESAMPLE with DML
2. TABLESAMPLE with some other percent than 0 and 100. to make the result consistent you can use the REPEATABLE clause.

Reply:
> to make the result consistent you can use the REPEATABLE clause.

consistent with what?

Reply:
> TABLESAMPLE with DML (either erroring or working)

added

Reply:
> TABLESAMPLE with some other percent than 0 and 100. to make the result consistent you can use the REPEATABLE clause.

added